src/logic/SolidLogic.ts

```diff
@@ -111,7 +113,7 @@ export class SolidLogic {
 
         // //// Load preference file
         try {
-        await this.store.fetcher.load(preferencesFile as NamedNode, {
+        await solidLogicSingleton.store.fetcher.load(preferencesFile as NamedNode, { // @@ was this.store.
```

Comment:
@timbl why was the replacement of this necessary here? 
If it does not work without it, we need to refactor the code otherwise we have a cyclic dependency here...

Reply:
No that can go back to

Reply:
Thank you! This spares me of some headaches :) and worries. I shall commit this part of code back.